src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +103,52 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search.
```

Comment:
If a single sample is represented across many read group sets one would like to return just those read group IDs for constructing the `readGroupIds` field of a `SearchReadsRequest`. Both fields are optional and only readgroups matching both will be returned.